libs/api/domains/user-profile/src/lib/dto/userDeviceTokenInput.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
+
+@InputType()
+export class UserDeviceTokenInput {
+  @Field(() => String)
+  @IsString()
```

Comment:
nit-picky but an unnecessary decorator ... doesn't do anything for the GraphQL schema.

Reply:
addressed in latest commit